entries/error.xml

```diff
@@ -34,18 +34,19 @@ $( "#book" )
   })
   .attr( "src", "missing.png" );
     </code></pre>
-    <p>If the image cannot be loaded (for example, because it is not present at the supplied URL), the alert  is displayed:</p>
+    <p>If the image cannot be loaded (for example, because it is not present at the supplied URL), the alert is displayed:</p>
     <p>
       <samp>Handler for .error() called.</samp>
     </p>
     <div class="warning">
-      <p>The event handler <em>must</em> be attached before the browser fires the error event, which is why the example sets the src attribute after attaching the handler. Also, the error event may not be correctly fired when the page is served locally; <code>error</code> relies on HTTP status codes and will generally not be triggered if the URL uses the <code>file:</code> protocol.</p>
+      <p>The event handler <em>must</em> be attached before the browser fires the <code>error</code> event, which is why the example sets the <code>src</code> attribute after attaching the handler. Also, the <code>error</code> event may not be correctly fired when the page is served locally; <code>error</code> relies on HTTP status codes and will generally not be triggered if the URL uses the <code>file:</code> protocol.</p>
     </div>
-    <p>Note: A jQuery error event handler should not be attached to the window object. The browser fires the window's error event when a script error occurs. However, the window error event receives different arguments and has different return value requirements than conventional event handlers. Use <code>window.onerror</code> instead.</p>
+    <p>Note: A jQuery <code>error</code> event handler should not be attached to the <code>window</code> object. The browser fires the <code>window</code>'s <code>error</code> event when a script error occurs. However, the window <code>error</code> event receives different arguments and has different return value requirements than conventional event handlers. Use <code>window.onerror</code> instead.</p>
   </longdesc>
   <example>
-    <desc>To hide the "broken image" icons for IE users, you can try:</desc>
+    <desc>To hide the "broken image" icons for users of older versions of IE, you can write:</desc>
```

Comment:
I remember is IE <= 10 but I can't find any reference. Someone with a hint on this?